### PR TITLE
Bjorncs/config convergence checker

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by hmusum — APPROVED** [review]
Great! 💯

File: configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java [hunk 1]
```diff
@@ -108,46 +117,67 @@ public JSONResponse getServiceConfigGenerationResponse(Application application,
...
+            if (statePort <= 0) continue;
+
+            URI uri = URI.create("http://" + service.getHostName() + ":" + statePort);
+            CompletableFuture<Void> inprogressRequest = getServiceGeneration(uri, timeout)
+                    .handle((result, error) -> {
+                        if (result != null) {
+                            temporaryResult.put(service, result);
+                        } else {
+                            log.log(
+                                    LogLevel.DEBUG,
```
Comment (jonmv) [inline configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java]: `Level.FINE`

**Comment by jonmv** [comment]
Nice :)

File: configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java (hunk 1 above)
Comment (bjorncs) [reply configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java]: I think they are equivalent (`LogLevel` being the Vespa wrapper for `Level`). Any benefits using `Level` instead?

Comment (jonmv) [reply configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java]: I know @gjoranv went to great lengths to replace with `Level` values some time ago. I don't know why, though.

Comment (bjorncs) [reply configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java]: Ok, I removed use of `LogLevel` in new PR.

Comment (gjoranv) [reply configserver/src/main/java/com/yahoo/vespa/config/server/application/ConfigConvergenceChecker.java]: I had to remove it from config in order to use config inside `vespalog`, and then it was proposed to remove `LogLevel` usage from all modules. I think the motivation was that custom log levels are unnecessary and a bit odd.

**Pull request merged**